faster_sam/adapter.py

```diff
@@ -92,6 +179,17 @@ def lambda_mapper(self, gateway_id: Optional[str]) -> Dict[str, Any]:
         return routes
 
     def register_routes(self, app: FastAPI, routes: Dict[str, Any]) -> None:
+        """
+        Registers FastAPI routes based on the provided routes dictionary.
```

Comment:
> Registers FastAPI routes based on the provided routes dictionary.

Registers ~FastAPI~ routes based on the provided _route map into the given FastAPI application_.